website/profile/views.py

```diff
@@ -382,6 +383,39 @@ def user_notifications(auth, **kwargs):
         'mailing_lists': auth.user.mailing_lists
     }
 
+@dev_only
+@must_be_logged_in
+def oauth_application_list(auth, **kwargs):
+    """Return app creation page with list of known apps"""
+    # TODO: Hardcoded URL
```

Comment:
Fair enough, and pinning it to apiv2 base discourages the main change I was worried about. Made change locally and will push once all review and responses collected.